src/tool/cppwinrt/strings/base_fast_fwd.h

```diff
@@ -0,0 +1,2123 @@
+
+namespace winrt::impl
+{
+// Thunk definitions in arch-specific assembly sources
+#define FAST_FWD_THUNK(index) extern "C" void __stdcall fast_fwd_thunk##index();
```

Comment:
Can we call it `WINRT_XXX` for consistency with all the other cppwinrt macros?

Reply:
yes, WINRT_FF_THUNK